sdk/communication/ci.yml

```diff
@@ -38,6 +38,8 @@ extends:
       safeName: AzureCommunicationSms
     - name: Azure.Communication.Identity
       safeName: AzureCommunicationIdentity
+    - name: Azure.Communication.MediaComposition
+      safeName: AzureCommunicationMediaComposition
```

Comment:
Since you are not planning to do a public release, consider adding `skipReleaseStage: true` parameter so that the release is never triggered.

Reply:
Added in my latest commit, thank you for the suggestion!